GitVersion.yml

```diff
@@ -0,0 +1,12 @@
+next-version: 1.0.3
+mode: mainline 
```

Comment:
Can this be simplified so that it's just `mode: mainline` ?

Reply:
File cut down to just `mode: mainline`